modules/installation-vsphere-installer-infra-requirements.adoc

```diff
@@ -25,30 +25,167 @@ Before you install an {product-title} cluster on your vCenter that uses infrastr
 [id="installation-vsphere-installer-infra-requirements-account_{context}"]
 == Required vCenter account privileges
 
-To install an {product-title} cluster in a vCenter, the installation program requires access to an account with privileges to read and create the required resources. Using an account that has administrative privileges is the simplest way to access all of the necessary permissions.
-
-A user requires the following privileges to install an {product-title} cluster:
-
-* Datastore
-** *Allocate space*
-** *Browse datastore*
-** *Low level file operations*
-** *Remove file*
-* Folder
-** *Create folder*
-** *Delete folder*
-* vSphere Tagging
-** All privileges
-* Network
-** *Assign network*
-* Resource
-** *Assign virtual machine to resource pool*
-* Profile-driven storage
-** All privileges
-* vApp
-** All privileges
-* Virtual machine
-** All privileges
+To install an {product-title} cluster in a vCenter, the installation program requires access to an account with privileges to read and create the required resources. Using an account that has global administrative privileges is the simplest way to access all of the necessary permissions.
+
+If you cannot use an account with global adminstrative privileges, you must create roles to grant the privileges necessary for {product-title} cluster installation. While most of the privileges are always required, some are required only if you plan for the installation program to provision a folder to contain the {product-title} cluster on your vCenter instance, which is the default behavior. You must create or amend vSphere roles for the specified objects to grant the required privileges.
+
+An additional role is required if the installation program is to create a vSphere virtual machine folder.
+
+.Roles and privileges required for installation
+[%collapsible]
+====
+[cols="3a,3a,3a",options="header"]
+|===
+|vSphere object for role
+|When required
+|Required privileges
+
+.10+|vSphere vCenter
+.10+|Always
+|`Cns.Searchable`
```

Comment:
Just wondering if there was a reason that these needed to each be in their own cell, as opposed to in a single cell, listed (or in bullets)? I think that complex tables aren't great for accessibility, and this seems like an instance where a complex table isn't necessarily needed.

It might also take up less vertical space if they aren't in individual cells.

Reply:
Bulleted lists take up about the same amount of space as the complex table. Forcing each item onto a separate line within the table cell does take up less space.

Reply:
Yeah when I said a bulleted list I meant the bullets being inside a single cell, not a bulleted list in place of the table entirely. table++ here. And I like the look a lot better after your update, thanks!

Reply:
I thought that's what you meant! Even *that* took up about the same amount of space because of the extra padding on bullets. :/